erpnext/hr/doctype/employee/employee.py

```diff
@@ -154,13 +159,13 @@ def validate_reports_to(self):
 			throw(_("Employee cannot report to himself."))
 
 	def on_trash(self):
+		self.update_nsm_model()
```

Comment:
Instead of calling `update_nsm_model`, call super function (on_trash of NestedSet class)